### ISSUE TITLE
Naming/MethodName doesn't flag attr_reader / attr_writer / attr_accessor

### ISSUE THREAD

**Issue by dmolesUC** [rubocop-hq/rubocop#7320]
## Expected behavior

[Naming/MethodName](https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_naming.md#namingmethodname) should flag `attr_reader`, `attr_writer`, and `attr_accessor` for incorrect case.

## Actual behavior

Only methods defined with explicit `def` statements are flagged.

## Steps to reproduce the problem

Given the following code:

```ruby
# Foo example
class Foo
  attr_accessor :fooBar
  attr_reader :barBaz
  attr_writer :bazQux

  def quxCorge
    'quxCorge'
  end

  def quxCorge=(_)
    raise 'not a thing'
  end
end
```

The `Naming/MethodName` cop should flag all five declarations -- `fooBar`, `barBaz`, `bazQux`, `quxCorge`, and `quxCorge=`

Instead, only the explicit `def quxCorge` and `def quxCorge=` are flagged:

```
Inspecting 1 file
C

Offenses:

/tmp/naming.rb:10:7: C: Naming/MethodName: Use snake_case for method names.
  def quxCorge
      ^^^^^^^^
/tmp/naming.rb:14:7: C: Naming/MethodName: Use snake_case for method names.
  def quxCorge=(_)
      ^^^^^^^^^

1 file inspected, 2 offenses detected
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
$ [bundle exec] rubocop -V
0.74.0 (using Parser 2.6.3.0, running on ruby 2.5.1 x86_64-darwin18)
```

**Comment by Drenmi** [comment]
Good catch, @dmolesUC!

**Comment by Drenmi** [comment]
Still relevant.
